src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -375,14 +376,16 @@ def build_actions(
     group: Group,
     project: Project,
     text: str,
-    color: str,
     actions: Sequence[MessageAction] | None = None,
     identity: RpcIdentity | None = None,
-) -> tuple[Sequence[MessageAction], str, str]:
+) -> tuple[Sequence[MessageAction], str, bool]:
     """Having actions means a button will be shown on the Slack message e.g. ignore, resolve, assign."""
     if actions and identity:
         text = get_action_text(text, actions, identity)
-        return [], text, "_actioned_issue"
+        if features.has("organizations:slack-improvements", project.organization):
+            # if actions are taken, return True at the end to show the white circle emoji
+            return [], text, True
+        return [], text, False
 
     status = group.get_status()
 
```

Comment:
should we move these functions outside of this method to help clean up reading the current function? Could also help us test out those individual functions. I can help with this in a separate PR!